packages/open-payments/src/client/requests.test.ts

```diff
@@ -365,4 +365,171 @@ describe('requests', (): void => {
       ).rejects.toThrow(/Failed to validate OpenApi response/)
     })
   })
+
+  describe('delete', (): void => {
+    const axiosInstance = createAxiosInstance({
+      requestTimeoutMs: 0,
+      privateKey,
+      keyId
+    })
+    const baseUrl = 'http://localhost:1000'
+    const responseValidators = mockOpenApiResponseValidators()
+
+    beforeAll(() => {
+      jest.spyOn(axiosInstance, 'delete')
+    })
+
+    test('properly makes DELETE request', async (): Promise<void> => {
+      const status = 202
+
+      // https://github.com/nock/nock/issues/2200#issuecomment-1280957462
+      jest
+        .useFakeTimers({
+          doNotFake: [
+            'nextTick',
+            'setImmediate',
+            'clearImmediate',
+            'setInterval',
+            'clearInterval',
+            'setTimeout',
+            'clearTimeout'
+          ]
+        })
+        .setSystemTime(new Date())
```

Comment:
I think this is only necessary if/when we validate the `Signature-Input` header's `created`